admin/package.json

```diff
@@ -4,7 +4,7 @@
   "private": false,
   "license": "GPL-3.0",
   "scripts": {
-    "lint": "standard",
+    "lint": "standard && tsc",
```

Comment:
Why?  Now it does two things...

Reply:
as per other comment, will separate into two distinct script calls